cli/mender-artifact/write.go

```diff
@@ -70,7 +70,7 @@ func writeRootfs(c *cli.Context) error {
 	var h handlers.Composer
 	switch version {
 	case 1:
-		h = handlers.NewRootfsV1(c.String("file"))
+		return cli.NewExitError("Error: Mender-Artifact version 1 is deprecated", errArtifactUnsupportedVersion)
```

Comment:
I would say "not supported". "Deprecated" indicates that it's still possible to use it.